docs/serving/app-security/security-guard-quickstart.md

```diff
@@ -0,0 +1,174 @@
+
+=== "kn CLI"
+
+    ```
+    kn service create helloworld-go \
+        --image gcr.io/knative-samples/helloworld-go \
+        --env "TARGET=Secured World" \
+        --annotation features.knative.dev/queueproxy-podinfo=enabled \
+        --annotation qpoption.knative.dev/guard-activate=enable
+    ```
+
+After the Service has been created, Guard starts monitoring the Service Pods and all Events sent to the Service.
+
+## Example alerts
+
+1. Send an event with unexpected query string, for example:
+
+     ```bash
+     curl "http://helloworld-go.default.52.118.14.2.sslip.io?a=3"
+     ```
+
+     This returns an output similar to the following:
+
+     ```sh
+     Hello Secured World!
+     ```
+
+1. Check alerts:
+
+     ```bash
+     kubectl logs deployment/helloworld-go-00001-deployment queue-proxy|grep "SECURITY ALERT!"
+     ```
+
+     This returns an output similar to the following:
+
+     ```sh
+     ...SECURITY ALERT! HttpRequest: QueryString: KeyVal: Key a is not known...
+     ```
+
+1. Send an event with unexpected long url, for example:
+
+     ```bash
+     curl "http://helloworld-go.default.52.118.14.2.sslip.io/AAAAAAAAAAAAAAAA"
+     ```
+
+     This returns an output similar to the following:
+
+     ```sh
+     Hello Secured World!
+     ```
+
+1. Check alerts:
+
+     ```bash
+     kubectl logs deployment/helloworld-go-00001-deployment queue-proxy|grep "SECURITY ALERT!"
+     ```
+
+     This returns an output similar to the following:
+
+     ```sh
+     ...SECURITY ALERT! HttpRequest: Url: KeyVal: Letters: Counter out of Range: 16...
+     ```
+
+
+## Managing the security of the service
+
+Guard offers situational awareness by writing its alerts to the Service queue proxy log. You may observe the queue-proxy to see alerts.
+
+Security alerts appear in the guard log file and start with the string `SECURITY ALERT!`. The default setup of Guard is to allow any request or response and learn any new pattern after reporting it. When the Service is actively serving requests, it typically takes about 30 min for Guard to learn the patterns of the Service requests and responses and build corresponding micro-rules. After the initial learning period, Guard updates the micro-rules in the Service Guardian, following which, it sends alerts only when a change in behavior is detected.
+
+Note that in the default setup, Guard continues to learn any new behavior and therefore avoids reporting alerts repeatedly when the new behavior reoccurs. Correct security procedures should include reviewing any new behavior detected by Guard.
+
+Guard can also be configured to operate in other modes of operation, such as:
+
+* Move from auto learning to manual micro-rules management after the initial learning period
+* Block requests/responses when they do not conform to the micro-rules
+
+For more information or for troubleshooting help, see the [#security](https://knative.slack.com/archives/CBYV1E0TG) channel in Knative Slack.
```

Comment:
I'd also make this its own page in the left nav - it's not clear how this should be part of the quickstart when it seems to reference more advanced configurations that can be done by users